### PR TITLE
Fix link accessibilty issues

### PR DESCRIPTION
**Pull request by Charles-Gagnon**
Fixes https://github.com/microsoft/azuredatastudio/issues/15771

This does mean that ALL links in the app are now underlined. I don't think this should be an issue as VS Code doesn't tend to use that many links, but something to keep an eye on and selectively disable if we think it's appropriate. 

@radentonev Does this seem ok with you?

I don't know if we'll ever get context on why they hid the underlining to begin with, it's been like that since the very beginning.

I considered having it be fancier such as only showing the underline on hover - but opted to go for the simpler route for now. We can revisit if we get negative feedback about changing this. 

image

image

image

### CODE REVIEW

File: src/vs/workbench/browser/media/style.css [hunk 1]
```diff
@@ -76,9 +76,9 @@ body.web {
 	cursor: pointer;
 }
 
-.monaco-workbench a {
-	text-decoration: none;
-}
+/* .monaco-workbench a {
+	text-decoration: none; {{SQL CARBON EDIT}} Underline links
```
Comment (alanrenmsft) [inline src/vs/workbench/browser/media/style.css:80]: we usually would try not to touch the original content, maybe comment out this way?
```CSS
/* {{SQLCARBON EDIT}} Underline links
.monaco-workbench a {
	text-decoration: none;
}
```
*/

Comment (Charles-Gagnon) [reply src/vs/workbench/browser/media/style.css:80]: Any reason why? From the perspective of a merge conflict it's all going to just be a single conflict anyways so as long as there's a carbon edit somewhere that fulfills the needs from that point of view.

**Review by alanrenmsft — APPROVED** [review]

File: src/vs/workbench/browser/media/style.css (hunk 1 above)
Comment (Charles-Gagnon) [reply src/vs/workbench/browser/media/style.css:80]: And I went there just to indicate that it was that one specifically. Don't really care either way but curious if you had a specific reason for wanting it the other way

Comment (alanrenmsft) [reply src/vs/workbench/browser/media/style.css]: yeah, for this one probably it is ok.

**Pull request merged**